From xiphmont@gmail.com Thu Nov 30 12:40:23 2006
Message-ID: <806dafc20611301240m36451682jeaf7eb4a0595c872@mail.gmail.com>
Date: Thu, 30 Nov 2006 15:40:10 -0500
From: Monty Montgomery <xiphmont@gmail.com>
To: greg@kroah.com, linux-usb-devel@lists.sourceforge.com
Subject: ehci: correct harmless bracketing and whitespace errors
MIME-Version: 1.0

From: Monty Montgomery <xiphmont@gmail.com>

This patch corrects a [harmless] bracketing error and some misplaced
whitespace in a kernel-doc in ehci-sched.c entry.  It results in
equivalent functionality.

Signed-off-by: Monty Montgomery <xiphmont@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-sched.c |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

--- gregkh-2.6.orig/drivers/usb/host/ehci-sched.c
+++ gregkh-2.6/drivers/usb/host/ehci-sched.c
@@ -1507,7 +1507,7 @@ static void periodic_init_link_restore_f
 {
 	int i;
 
-	if(use_fstn){
+	if(use_fstn)
 		ehci->periodic_restore_fstn =
 			ehci_fstn_alloc (ehci, GFP_ATOMIC);
 
@@ -1517,14 +1517,13 @@ static void periodic_init_link_restore_f
 
 		for(i=0; i<ehci->periodic_size; i++){
 			union ehci_shadow *here = &ehci->pshadow [i];
-		__le32			*hw_p = &ehci->periodic [i];
+			__le32		  *hw_p = &ehci->periodic [i];
 
 			here->fstn = ehci->periodic_restore_fstn;
 			*hw_p = FSTN_NEXT (here->fstn->fstn_dma);
 			wmb ();
 
 		}
-		}
 	}
 }
 
@@ -1537,7 +1536,7 @@ static void periodic_init_link_restore_f
  * @ehci: pointer to ehci host controller device structure
  * @frame: frame number in shadow/hardware schedule
  * @hw_p: returns pointer to insertion point in hardware schedule
-		 */
+ */
 static union ehci_shadow *
 periodic_position_save_fstn(struct ehci_hcd *ehci,
 			     unsigned frame,
